Monika After Story/game/pong.rpy

```diff
@@ -430,28 +579,502 @@ init -1 python in mas_pong:
+        
+            $ pong_last_responce_id = 120 
+            
+            
+        #On expert difficulty
+        elif pong_difficulty_before <= 20:
+            if pong_last_responce_id == 130:
+                m 5hub "It feels good to win!"
+                m 5eua "Don't worry, I'm sure you will win again soon~"
+            else:
+                if loss_streak_counter > 1:
+                    m 2eub "I win another round!"
+                else:
+                    m 2eub "I win this round!"
+        
+            $ pong_last_responce_id = 130
+            
+            
+        #On extreme difficulty
+        else:
+            if pong_last_responce_id == 140:
+                m 2duu "You have done well to come this far, [player]."
+                m 4esb "I gave it everything I got, so don't feel too bad for losing from time to time."
+            else:
+                m 2hub "This time it's my win!"
+                m 5hub "Keep up, [player]!"
+        
+            $ pong_last_responce_id = 140
+        
     return
+    
+    
 
-# end dialogue shown when monika is the pong winner
-label mas_pong_dlg_winner_end:
-    m 4tku "I can't really get excited for a game this simple..."
-    m 1eua "At least we can still hang out with each other."
-    m 1hub "Ahaha!"
-    m 1eua "Thanks for letting me win, [player]."
-    m 1tku "Only elementary schoolers seriously lose at Pong, right?"
-    m 1hua "Ehehe~"
+# dialogtue shown right when monika loses
+label mas_pong_dlg_loser:
+    # adapts the dialog, depending on the difficulty, game length (bounces), games losses, wins and dialog response.
+    # the order of the dialog is crucial, the first matching condition is chosen, other possible dialog is cancelled in the process.
+    # todo: adapt dialog to affection
+    # todo: add randomized dialog
+    
+    $ monika_asks_to_go_easy = False
+    
+     
+    #Monika loses on purpose
+    if lose_on_purpose:
+        m 5hub "Ahahaha!"
+        m 1kua "Now we're even, [player]!"
+        $ lose_on_purpose = False
+        
+        
+    #Monika loses without hitting the ball
+    elif ball_paddle_bounces == 0:
+        if pong_last_responce_id == 1010: 
+            m 1rksdlb "Maybe I should try a bit harder..."
+        else:
+            m 1rksdlb "Ahaha, I guess I was a bit too slow there..."
+        
+        $ pong_last_responce_id = 1010
+              
+              
+    #Player starts playing seriously and wins after losing at least 3 times on purpose
+    elif instant_loss_streak_counter_before >= 3 and persistent.player_once_let_monika_win_on_purpose:
+        m 5tsu "So you are playing seriously now?"#
+        m 1huu "Let's find out how good you really are, [player]!"  
+        
+    
+    #Player wins after losing at least three times in a row
+    elif loss_streak_counter_before >= 3:
+        m 4eub "Congrats [player]!"#
+        m 2eua "I knew you would win a game after enough practise!"
```

Comment:
_Very clever,_ but I think the spelling you want in this instance is "practice".